lib/ui/auth/bear_in_love_illustration.dart

```diff
@@ -0,0 +1,72 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+
+import '../core/themes/app_colors.dart';
+
+
+class BearInLoveIllustration extends StatefulWidget {
+  const BearInLoveIllustration({super.key});
+
+  @override
+  State<BearInLoveIllustration> createState() => _BearInLoveIllustrationState();
+}
+
+class _BearInLoveIllustrationState extends State<BearInLoveIllustration>
+    with SingleTickerProviderStateMixin {
+  late final AnimationController _controller;
+
+  @override
+  void initState() {
+    super.initState();
+    _controller = AnimationController(
+      vsync: this,
+      duration: const Duration(milliseconds: 900),
+    )..repeat(reverse: true);
+  }
+
+  @override
+  void dispose() {
+    _controller.dispose();
+    super.dispose();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return SizedBox(
+      height: 140,
+      child: Stack(
+        alignment: Alignment.center,
+        children: [
+          Row(
+            mainAxisAlignment: MainAxisAlignment.center,
+            children: [
+              SvgPicture.asset(
+                'assets/illustrations/pink-bear.svg',
+                width: 72,
+              ),
```

Comment:
1. 경로를 상수로 관리하면 어떨까요?
- 'pink-bear.svg'는 여러 곳에서 사용되고 있는데 이를 쓸 때마다 하드코딩으로 진행한다면 오타가 날 확률도 높고 무엇보다 관리하기가 힘듭니다!
```
class AppAssets {
  static const String pinkBear = 'assets/illustrations/pink-bear.svg';
  static const String purpleBear = 'assets/illustrations/purple-bear.svg';
}

// 실제 UI 코드에서 사용
const SvgPicture.asset(AppAssets.pinkBear, width: 72);
```
이런식으로 asset등을 사용할때는 클래스를 만들고 경로를 변수화하여 관리하는 게 좋습니다.

2. const 사용
- 만약 const 사용이 가능하다면 const를 붙이는 것이 메모리 효율에 좋습니다. (화면이 리빌드 될 때, 해당 위젯을 새로 생성하지 않고 기존 값을 그대로 재사용할 수 있습니다.)
```suggestion
              const SvgPicture.asset(
                'assets/illustrations/pink-bear.svg',
                width: 72,
              ),
```